tests/simulate/test_lammps.py

```diff
@@ -0,0 +1,220 @@
+"""Unit tests for relentless.simulate.lammps."""
+import tempfile
+import unittest
+
+import lammps
+import numpy as np
+
+import relentless
+from ..potential.test_pair import LinPot
+
+class test_LAMMPS(unittest.TestCase):
+    """Unit tests for relentless.LAMMPS"""
+
+    def setUp(self):
+        self._tmp = tempfile.TemporaryDirectory()
+        self.directory = relentless.Directory(self._tmp.name)
+
+    #mock (NVT) ensemble and potential for testing
+    def ens_pot(self):
+        ens = relentless.Ensemble(T=2.0, V=relentless.Cube(L=10.0), N={'1':2,'2':3})
+        ens.P = 2.5
+
+        # setup potentials
+        pot = LinPot(ens.types,params=('m',))
+        for pair in pot.coeff:
+            pot.coeff[pair]['m'] = 2.0
+        pots = relentless.simulate.Potentials()
+        pots.pair.potentials.append(pot)
+        pots.pair.rmax = 10.0
+        pots.pair.num = 11
+
+        return (ens,pots)
+
+    def create_file(self):
+        file_ = self.directory.file('test.data')
+        with open(file_,'w') as f:
+            f.write(('LAMMPS test data\n'
+                     '\n'
+                     '5 atoms\n'
+                     '2 atom types\n'
+                     '\n'
+                     '0.0 10.0 xlo xhi\n'
+                     '0.0 10.0 ylo yhi\n'
+                     '0.0 10.0 zlo zhi\n'
```

Comment:
LAMMPS supports general box origins like this, but we made the choice in `TriclinicBox` to use boxes centered around (0,0,0), so could you shift all your coordinates by -5.0? Like,
```suggestion
                     '-5.0 5.0 xlo xhi\n'
                     '-5.0 5.0 ylo yhi\n'
                     '-5.0 5.0 zlo zhi\n'
```
And similarly in the Atoms section.